src/tables.ts

```diff
@@ -9,6 +9,138 @@ import { RuntimeError } from './util';
 
 const RESOLUTION = 16384;
 
+export const CommonMetrics = {
+  fontFamily: 'Bravura',
+  fontSize: 30,
+  fontWeight: 'normal',
+  fontStyle: 'normal',
+
+  Accidental: {
+    cautionary: {
+      fontSize: 28,
+    },
+  },
+
+  Annotation: {
+    fontFamily: 'Arial, sans-serif',
+    fontSize: 10,
+  },
+
+  Bend: {
+    fontFamily: 'Arial, sans-serif',
+    fontSize: 10,
+  },
+
+  ChordSymbol: {
+    fontFamily: 'Roboto Slab, Times, serif',
+    fontSize: 12,
+  },
+
+  FretHandFinger: {
+    fontFamily: 'Arial, sans-serif',
+    fontSize: 9,
+    fontWeight: 'bold',
+  },
+
+  PedalMarking: {
+    fontFamily: 'Times New Roman, serif',
+    fontSize: 12,
+    fontWeight: 'bold',
+    fontStyle: 'italic',
+  },
+
+  Repetition: {
+    fontFamily: 'Times New Roman, serif',
+    fontSize: 13,
+    fontWeight: 'bold',
+  },
+
+  Stave: {
+    fontFamily: 'Arial, sans-serif',
+    fontSize: 8,
+  },
+
+  StaveConnector: {
+    fontFamily: 'Times New Roman, serif',
+    fontSize: 16,
+  },
+
+  StaveLine: {
+    fontFamily: 'Arial, sans-serif',
+    fontSize: 10,
+  },
+
+  StaveSection: {
+    fontFamily: 'Arial, sans-serif',
+    fontSize: 10,
+    fontWeight: 'bold',
+  },
+
+  StaveTempo: {
+    fontFamily: 'Times New Roman, serif',
+    fontSize: 14,
+    fontWeight: 'bold',
+  },
+
+  StaveText: {
+    fontFamily: 'Times New Roman, serif',
+    fontSize: 16,
+  },
+
+  StaveTie: {
+    fontFamily: 'Arial, sans-serif',
+    fontSize: 10,
+  },
+
+  StringNumber: {
+    fontFamily: 'Arial, sans-serif',
+    fontSize: 10,
+    fontWeight: 'bold',
+  },
+
+  Strokes: {
+    text: {
+      fontFamily: 'Times New Roman, serif',
+      fontSize: 10,
+      fontStyle: 'italic',
+      fontWeight: 'bold',
+    },
+  },
```

Comment:
It's interesting that all other metrics have fontFamily, fontSize, fontStyle, fontWeight defined on the Element type. 

Why is `Strokes.text` special? Why can't those properties be defined under Strokes directly?